### DISCUSSION TITLE
rcue to replace rq

### DISCUSSION THREAD

**Discussion by mbarkhau** [cue-lang/cue#1372] [Show and tell]
I can't be the first to have done this.

```shell
$ cat <<EOF > ~/bin/rcue
#!/bin/sh
cue eval - --out json -e "$@" <&0
EOF
$ chmod +x ~/bin/rcue
```

Usage

```
cat metrics.json | rcue 'data.metrics["ctr"]'
{
    "id": "ctr",
    "name": "CTR",
    "description": "Click through rate: Percentage of impressions that were clicked",

```

**Comment by verdverm** [comment]
I've been working on https://github.com/hofstadter-io/cuetils which has some other operations. The `Pick` operation is similar to query.

You might find #165 (Query Proposal) interesting. It will make this type of thing more powerful

**Comment by myitcv** [comment]
@mbarkhau #165 will add many features which will make this an even more exciting prospect for `cmd/cue`. Once #165 lands we will very likely consider how to expose a `jq`/`rq`/whatever-like UX for exactly the reason that motivated you to wrap `cue eval`.
